src/components/messenger/list/rewards-tooltip/container.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { connectContainer } from '../../../../store/redux-container';
+import { RootState } from '../../../../store/reducer';
+import { rewardsTooltipClosed } from '../../../../store/rewards';
+
+import { RewardsTooltip } from '.';
+import { Container as RewardsItemContainer } from '../../../settings-menu/rewards-item/container';
+export interface PublicProperties {}
+
+export interface Properties extends PublicProperties {
+  meowPreviousDayInUSD: string;
+  isLoading: boolean;
+
+  rewardsTooltipClosed: () => void;
+}
+
+export class Container extends React.Component<Properties> {
+  static mapState(state: RootState): Partial<Properties> {
+    const { rewards } = state;
+    return {
+      meowPreviousDayInUSD: RewardsItemContainer.totalPriceInUSD(rewards.meowPreviousDay, rewards.meowInUSD),
+      isLoading: rewards.loading,
+    };
+  }
+
+  static mapActions(_props: Properties): Partial<Properties> {
+    return { rewardsTooltipClosed };
+  }
+
+  render() {
+    if (this.props.isLoading || this.props.meowPreviousDayInUSD === '$0.00') {
```

Comment:
Should we use `showRewardsTooltip` here instead of hard coding 0 value? Is the saga handling `showRewardsTooltip` in that way? I haven't checked but would seem thats what the boolean is handling?

Reply:
similar to below comment .., i didn't want to pass this to the actual component so did it here. not sure about moving this to the saga since we have an initial fetch/sync triggered for the rewards when we login..